deploy/azureDeploy.js

```diff
@@ -14,13 +15,18 @@ class AzureDeploy {
 
     Object.assign(
       this,
+      createEventsBindings,
       loginToAzure,
       cleanUpFunctions,
       CreateResourceGroupAndFunctionApp,
       createFunctions
     );
 
     this.hooks = {
+      'before:package:initialize': () => BbPromise.bind(this)
```

Comment:
Since this PR allows calling the "serverless package" command directly, it would definitely seem to make more sense to split that implementation from the deploy command.